discovery-provider/src/utils/db_session.py

```diff
@@ -3,11 +3,21 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 from flask import current_app, g
+from src.queries.search_config import set_search_similarity
 
 
 session_manager = None
 
 
+def on_init_db(db):
```

Comment:
actually one question on this - i vaguely remember (unsure though) something about having to set the similarity threshold on each query since it was not persistent. did you confirm that this threshold persists on repeated calls? eg pg default is 0.3, so setting it to some other value then make repeated calls and confirm the limit persisted

Reply:
Confirmed that this threshold (in the new setup as well) persists b/w requests

```
a = session.execute(sqlalchemy.text(
        f"SHOW pg_trgm.similarity_threshold"
    )).fetchall()
logger.warning(a)
```

added this line in each search query and verified it spit out 0.1

Reply:
beaut